src/Illuminate/Foundation/Console/ListenerMakeCommand.php

```diff
@@ -46,12 +46,12 @@ protected function buildClass($name)
     {
         $event = $this->option('event');
```

Comment:
Might be better to do it one-time here.

```suggestion
        $event = $this->option('event') ?? '';
```